templates/docs/meter/cfos-meter-charge.yaml

```diff
@@ -1,4 +1,7 @@
 type: template
 template: cfos-meter
 description: cFos PowerBrain
-usage: charger
```

Comment:
warum hat unser render Test da nicht gefailt? Fehlt was in der Fehlerbehandlung?

Reply:
Das ist die generierte Doc die eh noch nicht benutzt wird und Arbeit braucht, das ist dann aber abhängig von der Integration in die Doku

Reply:
Mir gehts darum, dass bei falscher usage egtl. ein Fehler vom Charger kommen müsste wenn der getestet wird. Das ist hier aber nicht der Fall- fehlt vmtl. im Charger Code und sollte abgefangen werden.

Reply:
Anders gesagt: irgendwo fangen wir hier einen Fehler nicht ab. Im cfos Charger ist kein usage drin. Mit Nutzung der Doku hat das m.E. nix zu tun.

Reply:
Hab mal checks dafür eingebaut